Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12-slim
+LABEL maintainer="m.shmygel@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
The `requirements.txt` file is missing from the project files. Ensure that this file is included in your project directory to allow the `pip install -r requirements.txt` command to work correctly.